src/99-appendices/15-inheritance-principle.md

```diff
@@ -0,0 +1,270 @@
+# Appendix XV: The Inheritance Principle
+
+The [Inheritance Principle](#inheritance-principle) specifies how it is
+not strictly necessary to have data and metadata files with identical names
+apart from the [file extension](#definitions).
+It is possible to have a single metadata file with its contents applicable
+to multiple data files;
+it is additionally possible to have one data file where there are multiple
+metadata files from which metadata are applicable.
+This Appendix provides the precise definition of how this principle operates
+and the consequences of such,
+as reference for both software implementations and advanced BIDS users.
+
+## Rules
+
+1.  Any metadata file (such as `.json`, `.bvec` or `.tsv`) MAY be defined at any directory level.
+
+1.  For a given data file, any metadata file is applicable to that data file if:
+    1.  It is stored at the same directory level or higher;
+    1.  The metadata and the data filenames possess the same suffix;
+    1.  The metadata filename does not include any entity absent from the data filename.
+
+1.  A metadata file MUST NOT have a filename that would be otherwise applicable
+    to some data file based on rules 2.b and 2.c but is made inapplicable based on its
+    location in the directory structure as per rule 2.a.
+
+1.  If, for a given data file, multiple metadata files satisfy criteria 2.a-c above:
+
+    1.  The set of applicable metadata files is ordered as follows:
+
+        1.  Within each level of the filesystem hierarchy:
+
+            1.  Obtain the list of applicable metadata files within that directory.
+
+            1.  Sort this list in order of fewest to most entities.
+
+            1.  There MUST NOT be multiple applicable metadata files that
+                possess the same number of entities.
+
+            1.  The set of entities in any filename within this list MUST be a
+                strict superset of those present in the filename that precedes
+                it within this list.
+
+        1.  These lists are concatenated in order of highest to lowest level in the
+            filesystem hierarchy.
```

Comment:
note that 'highest' vs 'lowest' is ambiguous, please consider suggestion I gave before: https://github.com/bids-standard/bids-specification/pull/1003#discussion_r940232599